### ISSUE TITLE
Syntax Error: Python 3.7

### ISSUE THREAD

**Issue by ErikPartridge** [Kaggle/kaggle-api#64]
```  
File "/usr/local/bin/kaggle", line 7, in <module>
    from kaggle.cli import main
  File "/usr/local/lib/python3.7/site-packages/kaggle/__init__.py", line 18, in <module>
    from kaggle.api.kaggle_api_extended import KaggleApi
  File "/usr/local/lib/python3.7/site-packages/kaggle/api/__init__.py", line 22, in <module>
    from kaggle.api.kaggle_api_extended import KaggleApi
  File "/usr/local/lib/python3.7/site-packages/kaggle/api/kaggle_api_extended.py", line 27, in <module>
    from ..api_client import ApiClient
  File "/usr/local/lib/python3.7/site-packages/kaggle/api_client.py", line 289
    response_type=None, auth_settings=None, async=None,
                                                ^
SyntaxError: invalid syntax
```
I think this is coming from the use of async as a parameter name considering that it's a keyword since 3.5, but I may be wrong

**Comment by james97** [comment]
`async` is a reserved keyword from 3.7. I use python3.6 to create a virtual environment and it works

**Comment by rysteboe** [comment]
Fixed in 1.4.1.
